runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -93,11 +93,13 @@ J9::Z::CodeGenerator::initialize()
       cg->setSupportsInlineStringIndexOf();
       }
 
-   if (cg->getSupportsVectorRegisters() && !comp->getOption(TR_DisableSIMDStringHashCode) &&
-       !TR::Compiler->om.canGenerateArraylets() && !TR::Compiler->om.isOffHeapAllocationEnabled())
+   if (cg->getSupportsVectorRegisters() && !comp->getOption(TR_DisableSIMDStringHashCode) && !TR::Compiler->om.canGenerateArraylets())
       {
       cg->setSupportsInlineStringHashCode();
```

Comment:
Can you explain bit more here - Initially both String.hashCode and VectorizedHashCode was disabled for offHeap - This one enables String.hashCode but not other. Is that something we will be doing later - In that case, please add TODO comment here.

Reply:
I disabled it for testing; I have just enabled it again in the PR.